cds_migrator_kit/rdm/migration/transform/xml_processing/rules/base.py

```diff
@@ -121,23 +197,45 @@ def languages(self, key, value):
         raise UnexpectedValue(field=key, subfield="a")
 
 
-@model.over("subjects", "^693__")
+@model.over("subjects", "(^693__)|(^650[1_][7_])|(^653[1_]_)")
 @require(["a"])
 @filter_list_values
 def subjects(self, key, value):
-    """Translates languages fields."""
+    """Translates subjects fields."""
     _subjects = self.get("subjects", [])
-    subject_a = value.get("a")
-    subject_e = value.get("e")
-
-    if subject_a:
-        obj = {"subject": subject_a}
-        if obj not in _subjects:
-            _subjects.append(obj)
-    if subject_e:
-        obj = {"subject": subject_e}
-        if obj not in _subjects:
-            _subjects.append(obj)
+
+    if key == "693__":
+        subject_a = value.get("a")
+        subject_e = value.get("e")
+
+        if subject_a:
+            obj = {"subject": subject_a}
+            if obj not in _subjects:
+                _subjects.append(obj)
+        if subject_e:
+            obj = {"subject": subject_e}
+            if obj not in _subjects:
+                _subjects.append(obj)
+
+    if key == "65017":
+        subject_value = value.get("a")
+        subject = {
```

Comment:
how will this be handled later on? if I remember correctly, subjects are supposed to be a vocabulary, but how we will make rdm aware of these values?